docs/update-help-messages.sh

```diff
@@ -114,10 +114,10 @@ function parse_help_output() {
 
         # Parse subcommands that are followed by a section header. For example,
         # there are the subcommands under pgcopydb strem sentinel section:
-        #     create  Create the sentinel table on the source database
-        #     drop    Drop the sentinel table on the source database
-        #     get     Get the sentinel table values on the source database
-        #   + set     Maintain a sentinel table on the source database
+        #     create  Create the sentinel table
+        #     drop    Drop the sentinel table
```

Comment:
I'm confused as of why we still have these.

Reply:
You are right, fixed. 👍